### PR TITLE
Move node-sass to peerDependencies

### PR DESCRIPTION
**Pull request by jgerigmeyer**
Tests should run against the same version of node-sass that a project is compiled with. This forces users to have node-sass installed in order to use true, instead of using a true-specific version.

### CODE REVIEW

**Comment by mirisuzanne** [comment]
I wish we could hook it up to *any* Sass implementation they are using. With Ruby Sass on the way out, maybe we could at least get Dart Sass support in soon?

**Comment by jgerigmeyer** [comment]
I think that'd be possible, if we make both `node-sass` and `sass` (dart-sass) peerDependencies. Users would get a warning though, assuming they only have one or the other installed... We could go farther and make them both `optionalDependencies`, but then I'd want to be clear in the docs that you need one or the other.

**Comment by jgerigmeyer** [comment]
Oh, nevermind. `optionalDependencies` would defeat the purpose entirely. But `peerDependencies` could work, if we don't mind some warnings on install.

File: README.md [hunk 1]
```diff
@@ -36,7 +37,7 @@ In command line:
 
 ```bash
 # npm module
-npm install sass-true
+npm install sass-true node-sass
```
Comment (jgerigmeyer) [inline README.md]: I'm not sure if this requires more explanation or not? Most users will probably already have `node-sass` installed?

Comment (mirisuzanne) [reply README.md]: I think it's worth explaining for the people that don't have it.

**Pull request merged**